src/py/flwr/server/app.py

```diff
@@ -390,6 +419,61 @@ def run_superlink() -> None:
         driver_server.wait_for_termination(timeout=1)
 
 
+def _try_setup_client_authentication(
+    args: argparse.Namespace,
+    certificates: Optional[Tuple[bytes, bytes, bytes]],
+) -> Optional[Tuple[Set[bytes], ec.EllipticCurvePublicKey, ec.EllipticCurvePrivateKey]]:
+    if not args.require_client_authentication:
+        return None
+
+    if certificates is None:
+        sys.exit(
+            "Certificates are required to enable client authentication. "
+            "Please provide certificate paths with '--certificates' before "
+            "enabling '--require-client-authentication'."
+        )
+
+    client_keys_file_path = Path(args.require_client_authentication[0])
+    if not client_keys_file_path.exists():
+        sys.exit(
+            "Client public keys csv file are required for client authentication. "
+            "Please provide the csv file path containing known client public keys "
+            "to '--require-client-authentication'."
```

Comment:
```suggestion
            "The provided path to the client public keys CSV file does not exist. "
            "Please provide the CSV file path containing known client public keys "
            "to '--require-client-authentication'."
```

Reply:
We could also echo the path here to make it easier for users to see which path caused the error